packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
- * Copyright (C) 2018 Wire Swiss GmbH
+ * Copyright (C) 2022 Wire Swiss GmbH
```

Comment:
```suggestion
 * Copyright (C) 2018 Wire Swiss GmbH
```

Only files created in 2022 need to have 2022 in the header :)